fixtures/valid-questionnaire-template.js

```diff
@@ -6828,6 +6828,36 @@ const validQTemplate = {
                 ]
             }
         },
+        user: {
+            schema: {
+                $schema: 'http://json-schema.org/draft-07/schema#',
+                type: 'object',
+                required: ['user-id'],
+                properties: {
+                    'user-id': {
+                        type: 'string',
+                        pattern: '^urn:([a-z.]*:){2}[0-9]*:[0-9A-Za-z=_-]{44}$',
```

Comment:
My username fails this validation as the last section only has 43 characters:

`urn:fdc:gov.uk:2022:ZoTyx0owL1MYS-UkCwtQXbF2A-padOhdssGvXDfamws`